exporter/clients.go

```diff
@@ -28,15 +29,27 @@ func parseClientListString(clientInfo string) ([]string, bool) {
 		connectedClient[vPart[0]] = vPart[1]
 	}
 
+	createdAt, err := durationFieldToTimestamp(connectedClient["age"])
+	if err != nil {
+		log.Debugf("cloud not parse age field(%s): %s", connectedClient["age"], err.Error())
+		return nil, false
+	}
+
+	idleSince, err := durationFieldToTimestamp(connectedClient["idle"])
+	if err != nil {
+		log.Debugf("cloud not parse idle field(%s): %s", connectedClient["idle"], err.Error())
+		return nil, false
+	}
+
 	hostPortString := strings.Split(connectedClient["addr"], ":")
 	if len(hostPortString) != 2 {
 		return nil, false
 	}
 
 	return []string{
 		connectedClient["name"],
-		connectedClient["age"],
-		connectedClient["idle"],
+		strconv.FormatInt(createdAt, 10),
```

Comment:
Can you use `strconv.Itoa()` ? And call that in the `durationFieldToTimestamp()` helper function?

Reply:
Itoa accepts int, not int64 which is required to prevent int32 overflow in the years to follow.

Reply:
purpose of `durationFieldToTimestamp` is to convert duration to timestamp, not to string. It makes sense to do FormatInt in the parseClientListString, because the logic of createdAt and idleSince being string is related to `parseClientListString`, not `durationFieldToTimestamp`

Reply:
Ah right, int64 compatibility is a good call. Still, can you do the call to `strconv.FormatInt()` inside durationFieldToTimestamp() and return a string instead of int? In that way you can also get rid of the helper function in the tests.